src/java/org/apache/cassandra/transport/Dispatcher.java

```diff
@@ -154,14 +323,49 @@ public String description()
         {
             return request.toString();
         }
+
+        @Override
+        public String toString()
+        {
+            return "RequestProcessor{" +
+                   "request=" + request +
```

Comment:
nit: `Message` has a `toString()`, but `Request` does not, so we'll miss `Request.createdAtNanos`...if that matters.